fallback-config.nix.mustache

```diff
@@ -0,0 +1,93 @@
+# This file was generated from `meta.yml`, please do not edit manually.
+# Follow the instructions on https://github.com/coq-community/templates to regenerate.
+
+# *However*, you may copy paste it to `config.nix`
+# in the same subdirectory .nix and extend it as needed.
+# Note that this would completely deactivate this file.
+
+{
+  ## DO NOT CHANGE THE format
+  format = "1.0.0";
+  ## unless you made an automated or manual update
+  ## to another supported format.
+
+  ## The attribute to build, either from nixpkgs
+  ## of from the overlays located in `.nix/coq-overlays`
+  attribute = "{{nix_name}}{{^nix_name}}{{ shortname }}{{/nix_name}}";
+
+  ## If you want to select a different attribute
+  ## to serve as a basis for nix-shell edit this
+  # shell-attribute = "{{nix_name}}{{^nix_name}}{{ shortname }}{{/nix_name}}";
+
+  ## Maybe the shortname of the library is different from
+  ## the name of the nixpkgs attribute, if so, set it here:
+  {{^ shortname}}#{{/ shortname}}pname = "{{ shortname }}";
+
+  ## Lists the dependencies, phrased in terms of nix attributes.
+  ## No need to list Coq, it is already included.
+  ## These dependencies will systematically be added to the currently
+  ## known dependencies, if any more than Coq.
+  ## /!\ Remove this field as soon as the package is available on nixpkgs.
+  ## /!\ Manual overlays in `.nix/coq-overlays` should be preferred then.
+  {{^ nix-dependencies}}#{{/ nix-dependencies}}buildInputs = [ {{# nix-dependencies }}"{{.}}" {{/ nix-dependencies }}];
+
+  ## Indicate the relative location of your _CoqProject
+  ## If not specified, it defaults to "_CoqProject"
+  coqproject = "{{coqproject}}{{^coqproject}}_CoqProject"{{/coqproject}};
+
+  ## Select a task to build by default in the following `tasks` set
+  ## It will be used when typing nix-shell or nix-build without argument
+  ## defaults to "default"
+  default-task = "{{nix-default-task}}{{^nix-default-task}}default{{/nix-default-task}}";
+  tasks."{{nix-default-task}}{{^nix-default-task}}default{{/nix-default-task}}" = {};
+
+{{#nix-tasks}}
+  tasks."{{task}}" = {
+{{#packages}}
+{{#version}}
+      {{location}}{{^location}}coqPackages.{{/location}}"{{name}}".override.version = "{{version}}";
+{{/version}}
+{{#job}}
+      {{location}}{{^location}}coqPackages.{{/location}}"{{name}}".job = "{{job}}";
+{{/job}}
+{{/packages}}
+  };
+{{/nix-tasks}}
+{{^nix-tasks}}
+  ## write one `tasks.name` attribute set per
+  ## alternative configuration, they can be used to
+  ## compute several ci jobs as well
+  tasks.default = {
+
+  ## You can override Coq and other Coq coqPackages
+  ## through the following attribute
+  # coqPackages.coq.override.version = "8.11";
```

Comment:
I keep thinking that `fallback-config.yml` could be removed in favor of a `config.yml` that is entirely generated from `meta.yml` but for this, it is important to be able to set this value.